apps/asap-server/test/handlers/users/fetch.test.ts

```diff
@@ -100,8 +98,8 @@ describe('GET /users', () => {
           Authorization: `Bearer token`,
         },
         queryStringParameters: {
-          take: 8,
-          skip: 8,
+          take: '8',
```

Comment:
Why a string?

Reply:
QueryStringParameters will never be a string https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/aws-lambda/trigger/api-gateway-proxy.d.ts#L107

Reply:
If Joi parses it as number I don't see a problem :+1:

Reply:
These are not the queryStringParameters the FE sends us. This are the ones that are presented to the lambda framework after AWS processes them. These will always be strings and it's up to Joi to then convert them to numbers then. Them not being strings is actually wrong.